app/src/organisms/Desktop/Devices/ProtocolRun/ProtocolRunHeader/RunHeaderContent/RunHeaderSectionUpper.tsx

```diff
@@ -74,7 +85,19 @@ export function RunHeaderSectionUpper(
             {t('visualize')}
           </SecondaryButton>
         ) : null}
-        <ActionButton {...props} />
+        <Flex gridGap={SPACING.spacing8}>
+          {isCancellableStatus(runStatus) && (
+            <PrimaryButton
+              borderRadius="200px"
```

Comment:
```suggestion
              borderRadius={BORDERS.borderRadiusFull}
```
Almost, there's a variable associated with that property in figma.